docs/basic-remote-setup.md

```diff
@@ -35,16 +35,19 @@ clients:
     insecure: yes
     trusted: yes
     redirect_uris:
-      - http://localhost:9100/oidc-callback.html
       - http://localhost:9100/
+      - http://localhost:9100/oidc-callback.html
+      - http://localhost:9100/oidc-silent-redirect.html
       - https://localhost:9200/
       - https://localhost:9200/oidc-callback.html
+      - https://localhost:9200/oidc-silent-redirect.html
       - https://your-host:9200/
       - https://your-host:9200/oidc-callback.html
+      - https://your-host:9200/oidc-silent-redirect.html
```

Comment:
Please change all the `your-host`s to `your-server` in the `redirect_uris` and `origins` as well (like you did for the env vars)

Reply:
done